Cargo.toml

```diff
@@ -31,6 +31,8 @@ log = "0.4"
 env_logger = "0.10"
 human-panic = "1.0"
 chrono = "0.4.30"
+chrono-tz = "0.8.3"
+regex = "1.9.5"
```

Comment:
This is only used in tests, so it should live under `dev-dependencies` 🙂

Reply:
Ah, thanks -- sorry about that!